apps/autoscheduler/FunctionDAG.cpp

```diff
@@ -601,12 +601,11 @@ FunctionDAG::FunctionDAG(const vector<Function> &outputs, const MachineParams &p
 
         // Create a symbolic region for this Func.
         for (int j = 0; j < consumer.dimensions(); j++) {
-            Expr min_var = Variable::make(Int(32), consumer.name() + "." + consumer.args()[j] + ".min");
-            Expr max_var = Variable::make(Int(32), consumer.name() + "." + consumer.args()[j] + ".max");
-            Expr extent = max_var - min_var + 1;
+            Halide::Var min_var(consumer.name() + "." + consumer.args()[j] + ".min");
+            Halide::Var max_var(consumer.name() + "." + consumer.args()[j] + ".max");
             Interval interval(min_var, max_var);
             scope.push(consumer.args()[j], interval);
-            node.region_required.push_back(interval);
+            node.region_required.push_back(SymbolicRange{min_var, max_var});
```

Comment:
Would this be better as `emplace_back()`?

Reply:
Indeed, fixed